File: Cpp/1006.cpp
#include <iostream>
#include <iomanip>
 
using namespace std;
 
int main() {
 
    double a, b, c, m;
    
    cin >> a;
    cin >> b;
    cin >> c;
    
    m = (2*a + 3*b + 5*c)/10;
    
    cout << fixed << setprecision(1);
    cout << "MEDIA = " << m << "\n";
 
    return 0;
}

File: Haskell/1006.hs
import Text.Printf ( printf )

main ::  IO ()
main = do
    a <- readLn :: IO Double
    b <- readLn :: IO Double
    c <- readLn :: IO Double
    let x = (2*a + 3*b + 5*c)/10
    printf "MEDIA = %.1f\n" x